src/Controller/EnMarche/UserController.php

```diff
@@ -33,6 +35,22 @@ public function profileOverviewAction(): Response
         return $this->render('user/my_account.html.twig');
     }
 
+    /**
+     * @Route("/mes-dons", name="app_enmarche_user_profiledonation")
+     * @Method("GET|POST")
```

Comment:
why is the `POST` method needed here?

Reply:
```
    /**
     * @Route("", name="app_user_profile")
     * @Method("GET|POST")
     */
```

Reply:
don t know why POST , but I have the same question for the route app_user_profile :D

Reply:
:thinking: yeap nice catch, this should also be fixed